osu.Game/Screens/Play/Player.cs

```diff
@@ -73,6 +74,8 @@ public class Player : ScreenWithBeatmapBackground
 
         protected HUDOverlay HUDOverlay { get; private set; }
 
+        protected HitErrorDisplayOverlay HitErrorDisplayOverlay { get; private set; }
```

Comment:
why is this no longer in the hud?

Reply:
I thought since it not depends on hud visibility, it should be moved from it

Reply:
It is part of the HUD and *should* be affected, IMO.

Reply:
Alright then, just using the stable for reference